dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -1,8 +1,7 @@
 apiVersion: v1
 data:
   rhtap-slo-dashboard.json: |-
-    {
-      "annotations": {
+    "annotations": {
```

Comment:
This change looks suspicious, it makes invalid JSON

Reply:
ty.